### DISCUSSION TITLE
Microsoft learn Account

### DISCUSSION THREAD

**Discussion by anzilparviz29** [microsoft/SATechnicalOnboarding#56] [Questions]
Hey iam new to Student Ambassador Program, whether i need to use my old learn microsoft account or the new one which microsoft had provided me ie, microsoft 365 account , for completing learning paths

**Comment by pabuce** [comment]
I think you should use the new one, in order to sync that data with your MLSA profile.

**Comment by anzilparviz29** [comment]
Thanks sir
